crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -447,6 +560,24 @@ impl ReadOnlyUserIdentity {
         &self.self_signing_key
     }
 
+    /// Pin the current identity
+    pub(crate) fn pin(&self) {
+        let mut m = self.pinned_msk.write().unwrap();
+        *m = self.master_key.as_ref().clone()
+    }
+
+    /// Key pinning acts as a trust on first use mechanism, the first time an
+    /// identity is known for a user it will be pinned.
+    /// For future interaction with a user, the identity is expected to be the
+    /// one that was pinned. In case of identity change the UI client should
+    /// receive reports of pinning violation and decide to act accordingly;
+    /// that is accept and pin the new identity, perform a verification or
+    /// stop communications.
```

Comment:
The first line should be a short explanation of the method kept under some 80-100 characters, then we can expand on the first line in the next paragraph.

Could you try to reword this to conform this style?